DEVELOPERS.md

```diff
@@ -93,7 +91,11 @@ If you need to upgrade an installation the [ArchWiki](https://wiki.archlinux.org
 You'll need a database in Postgres to work with, run:
 
 ```sh
-psql -c "create database jobserver"
+psql -c "CREATE DATABASE jobserver"
+psql -c "
+CREATE ROLE jobsuser PASSWORD 'pass' NOSUPERUSER CREATEDB;
+GRANT ALL PRIVILEGES on database jobserver to jobsuser;
+"
```

Comment:
We should probably find a better way to standardise this, I know OSI had a run at putting postgres inside docker-compose by default.  This works but isn't necessary on macOS when using Postgres.app, but I think this is fine to document until we have a better story in general!

Reply:
Hmm. I guess that's why it wasn't documented.  Do you also not need to create the user with Postgres.app?  I got a bunch of unexpected django postgres errors trying to run tests.  I could add that the create role/grant bit is only required on linux?

Reply:
Yeah Postgres.app sets up ident auth so your OS user gets the equivalent of these steps.  You could add these are only for linux for now and then we can tackle this probably later for all the sites :)